include/bluetooth/direction.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_DF_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_DF_H_
+
+/** @brief Constant Tone Extension parameters for connectionless
+ * transmission.
+ *
+ * The structure holds information required to setup CTE transmission
+ * in periodic advertising.
+ */
+struct bt_le_df_adv_cte_tx_params {
+	/* Length of CTE in 8us units */
+	uint8_t  cte_len;
+	/* CTE Type: AoA, AoD 1us slots, AoD 2us slots */
+	uint8_t  cte_type;
+	/* Number of CTE to transmit in each periodic adv interval */
+	uint8_t  cte_count;
+	/* Number of Antenna IDs in the switch pattern */
+	uint8_t  num_ant_ids;
+	/* List of antenna IDs in the pattern */
+	uint8_t  *ant_ids;
+};
+
+/**
+ * @brief Set or update the Constant Tone Extension parameters for periodic
+ * advertising set.
+ *
+ * The Constant Tone Extension parameters can only be set or updated on an
+ * extended advertisment set which has already set periodic adverising
+ * parameters. Update of Constant Tone Extension parameters may not be attempted
```

Comment:
The updated sentence is unclear what means here now.

Do I need to stop both the periodic advertiser (bt_le_per_adv_stop) and the extended advertiser (bt_le_ext_adv_stop) in order to be able to update the CTE parameters?

Suggest to reword "attempted". You could always attempt, but would return an error code. Should use "allowed" instead.

Reply:
I've corrected the description.
First part of it was my wrong understanding of specification. Parameters may be set on existing advertising set.
That is obvious because one has to provide bt_le_ext_adv reference.
Second part is related with function that will be provided in following PR. 
The function enables transmission of CTE with periodic advertising.
To update CTE parameters, transmission must be disabled before.

I've updated the descritpion.